README.md

```diff
@@ -27,22 +27,22 @@ This repository contains backend part of **Tags Drive**
 
 | Variable       | Default | Description                                                              |
 | -------------- | ------- | ------------------------------------------------------------------------ |
-| PORT           | 80      | Port for website                                                         |
-| TLS            | true    | Should **Tags Drive** use https                                          |
-| LOGIN          | user    | Login for login                                                          |
-| PSWRD          | qwerty  | Password for login                                                       |
-| ENCRYPT        | false   | Should the **Tags Drive** encrypt uploaded files                         |
+| PORT           | 80      | Assign your website’s port                                                        |
+| TLS            | true    | Enable HTTPS                                        |
+| LOGIN          | user    | Set your login                                                        |
+| PSWRD          | qwerty  | Set your password                                                     |
+| ENCRYPT        | false   | Encrypt uploaded files                         |
 | DBG            | false   |                                                                          |
-| SKIP_LOGIN     | false   | Let use **Tags Drive** without loginning                                 |
-| PASS_PHRASE    | ""      | Passphrase is used to encrypt files. It can't be empty if `ENCRYPT=true` |
-| MAX_TOKEN_LIFE | 1440h   | Max lifetime of a token (default is 60 days)                             |
+| SKIP_LOGIN     | false   | Skip the log-in procedure                                |
+| PASS_PHRASE    | ""      | Set a phrase for file encryption. Cannot be empty if `ENCRYPT == true` |
+| MAX_TOKEN_LIFE | 1440h   | Set the max lifetime of a token (default lifetime is 60 days)                             |
 
 ## Development
 
-There are two Python scripts to run a local version:
+There are two Python scripts that you can use to run a local version of the program:
```

Comment:
There are two Python scripts that you can use to run a local version of **the backend part**